tile.go

```diff
@@ -19,20 +23,39 @@ type Tile struct {
 	Extent    *float64
 }
 
-func (t *Tile) Deg2Num() (x, y int) {
+func (t *Tile) Deg2Num() (x, y int, err error) {
+	// Converts from WGS84 Lat/Long postion to tile position (x,y)
+
+	// Check that input lat/long values are within WGS84 bounds
+	if t.Lat < -85.0511 || t.Lat > 85.0511 || t.Long < -180.0 || t.Long > 180.0 {
+		msg := fmt.Sprintf("One or both outside valid range (Long, Lat): (%v, %v)", t.Long, t.Lat)
+		err = fmt.Errorf("%v", msg)
```

Comment:
No need to do fmt.Sprintf(…) and then fmt.Errorf(…), combine the two:
`err = fmt.Errorf("One or both outside valid range (Long, Lat): (%v, %v)", t.Long, t.Lat)`

Also, for errors, we are trying to standardize to starting with a lower letter for the message. i.e:
`err = fmt.Errorf("one or both are outside the valid range (Long, Lat): (%v, %v)", t.Long, t.Lat)`

Reply:
In my experience the calculations are not perfect and therefore can produce values that are outside the range. I would prefer to not log.Fatal right now until we have a bullet proof calculation. Maybe if the value is out of range we could return the max value that's in range?